packages/fast-development-site-react/src/components/site/configuration-panel.tsx

```diff
@@ -0,0 +1,213 @@
+    configurationPanel: {
+        width: "100%"
+    },
+    configurationPanel_controls: {
+        display: "flex",
+        padding: `${toPx(2)} ${toPx(4)}`,
+        "& ul": {
+            margin: "0",
+            padding: "0",
+            listStyleType: "none",
+            "& li": {
+                display: "inline-block"
+            }
+        },
+        "& button": {
+            border: "none",
+            height: toPx(38),
+            padding: `${toPx(6)} ${toPx(8)}`,
+            minWidth: toPx(25),
+            backgroundPosition: "center"
+        }
+    },
+    configurationPanel_controls_tabs: {
+        flexGrow: "1"
+    },
+    configurationPanel_tab: {
+        "& button": {
+            background: "none",
+            "&:focus": {
+                outline: "none"
+            }
+        }
+    },
+    configurationPanel_tab__active: {
+        "& button": {
+            borderRadius: toPx(3),
+            position: "relative",
+            "&:after": {
+                position: "absolute",
+                bottom: "0",
+                left: toPx(4),
+                right: toPx(4),
+                content: "''",
+                height: toPx(2),
+                borderRadius: toPx(3),
+                // TODO: Issue #309 https://github.com/Microsoft/fast-dna/issues/309
+                // background: (config: IDevSiteDesignSystem): string => {
+                //     return config.brandColor;
+                // }
+                background: "#FB356D"
+            }
+        }
+    },
+    configurationPanel_tabPanel: {
+        "&[aria-hidden=\"true\"]": {
+            display: "none"
+        }
+    },
+    configurationPanel_paneForm: {
+        padding: toPx(12)
+    },
+};
+
+/* tslint:disable-next-line */
+class ConfigurationPanel extends React.Component<IConfigurationPanelProps & IManagedClasses<IConfigurationPanelManagedClasses>, IConfigurationPanelState> {
+    private tabs: TabType[];
+
+    constructor(props: IConfigurationPanelProps & IManagedClasses<IConfigurationPanelManagedClasses>) {
+        super(props);
+
+        this.tabs = [TabType.presets];
+
+        this.state = {
+            activeTab: this.props.activeTab || TabType.presets
+        };
+    }
+
+    public render(): JSX.Element {
+        return (
+            <div className={this.props.managedClasses.configurationPanel}>
+                <div className={this.props.managedClasses.configurationPanel_controls}>
+                    {this.renderTabs()}
+                </div>
+                {this.renderTabPanels()}
+            </div>
+        );
+    }
+
+    private renderTabs(): JSX.Element {
+        return (
+            <ul className={this.props.managedClasses.configurationPanel_controls_tabs}>
+                {this.renderTabItems()}
+            </ul>
+        );
+    }
+
+    private renderTabItems(): JSX.Element[] {
+        return this.tabs.map((tabItem: TabType, index: number) => {
+            if (tabItem === "Presets") {
```

Comment:
is there any possibility for the "Presets" string to be lower case? If so protect against as well.

Reply:
Since we are expecting the TabType enum here we should likely check against the enum rather than a string.